package.json

```diff
@@ -44,16 +46,13 @@
     "eslint": "^5.14.1",
     "eslint-config-prettier": "^4.0.0",
     "eslint-plugin-prettier": "^3.0.1",
-    "fetch-mock": "^7.3.1",
     "husky": "^1.3.1",
     "jest": "^24.1.0",
     "lint-staged": "^8.1.4",
-    "node-fetch": "^2.3.0",
     "prettier": "^1.16.4",
     "prettier-eslint": "^8.8.2",
     "prettier-eslint-cli": "^4.7.1",
     "react": "^16.8.3",
-    "react-async": "^5.1.0",
```

Comment:
Whoops! these must have snuck into master when I was updating the README a few days ago. Nonetheless, they are not required.